### PR TITLE
chore: Workflow execution JS-SDK

### PR DESCRIPTION
**Pull request by olivermrbl**

### CODE REVIEW

File: packages/core/types/src/http/workflow-execution/admin/entities.ts [hunk 1]
```diff
@@ -1,11 +1,92 @@
+export type TransactionStepStatus =
```
Comment (olivermrbl) [inline packages/core/types/src/http/workflow-execution/admin/entities.ts:1]: I got most of these types from admin, but not sure where to check to validate if they are correct

**Review by sradevski — APPROVED** [review]
LGTM! For the typings, I guess you already did this, but we can maybe check `workflow-engine-*` and check the response types there?

**Comment by olivermrbl** [comment]
> LGTM! For the typings, I guess you already did this, but we can maybe check `workflow-engine-*` and check the response types there?

Yeah, I tried to cross-check, but many of the types in those packages are just `Record<string, unknown`s. I'll take one more pass at this and then merge.

File: packages/core/types/src/http/workflow-execution/admin/entities.ts (hunk 1 above)
Comment (olivermrbl) [inline packages/core/types/src/http/workflow-execution/admin/entities.ts:1]: @carlos-r-l-rodrigues, perhaps you could glance over these and let me know if something is off.

**Review by fPolic — APPROVED** [review]

File: packages/core/types/src/http/workflow-execution/admin/entities.ts (hunk 1 above)
Comment (carlos-r-l-rodrigues) [reply packages/core/types/src/http/workflow-execution/admin/entities.ts:1]: It looks great. Thanks!

**Pull request merged**